emails/utils.py

```diff
@@ -346,31 +344,6 @@ def get_reply_to_address(premium: bool = True) -> str:
     return reply_to_address
 
 
-def generate_relay_From(
-    original_from_address: str, user_profile: Profile | None = None
-) -> str:
-    # RFC 2822 (https://tools.ietf.org/html/rfc2822#section-2.1.1)
-    # says email header lines must not be more than 998 chars long.
-    # Encoding display names to longer than 998 chars will add wrap
-    # characters which are unsafe. (See https://bugs.python.org/issue39073)
-    # So, truncate the original sender to 900 chars so we can add our
-    # "[via Relay] <relayfrom>" and encode it all.
-    if len(original_from_address) > 998:
-        original_from_address = "%s ..." % original_from_address[:900]
-    # line breaks in From: will encode to unsafe chars, so strip them.
-    original_from_address = (
-        original_from_address.replace("\u2028", "").replace("\r", "").replace("\n", "")
-    )
-
-    display_name = Header('"%s [via Relay]"' % (original_from_address), "UTF-8")
-    user_has_premium = bool(user_profile and user_profile.has_premium)
-    relay_from_address = get_reply_to_address(user_has_premium)
-    formatted_from_address = str(
-        Address(display_name.encode(maxlinelen=998), addr_spec=relay_from_address)
-    )
-    return formatted_from_address
-
-
```

Comment:
change (blocking, though not part of this particular code change): When I looked for the corresponding code in the new `generate_from_header` function, I saw this:
```
      # Truncate the to 71 characters, so the sender portion fits on the first
```

Looks like it should say "Truncate the display_name to 71 characters ..."?